templates/galaxy/config/object_store_conf.yml.j2

```diff
@@ -0,0 +1,289 @@
+type: distributed
+##global_max_percent_full: 90
+search_for_missing: true
+id: primary     # is that needed
+order: 0        # is that needed
+backends:
+  - id: secondary
+    type: disk
+    store_by: id
+    weight: 0
+    files_dir: "/data/0/galaxy_db/files"
+
+  - id: files1
+    type: disk
+    store_by: id
+    name: "University of Freiburg storage (NetApp FabricPool, id=files1)"
+    weight: 0
+    files_dir: "/data/1/galaxy_db/files"
+
+  - id: files2
+    type: disk
+    store_by: id
+    name: "University of Freiburg storage (NetApp FabricPool, id=files2)"
+    weight: 0
+    files_dir: "/data/2/galaxy_db/files"
+
+  - id: files3
+    type: disk
+    store_by: id
+    name: "University of Freiburg storage (NetApp FabricPool, id=files3)"
+    weight: 0
+    files_dir: "/data/3/galaxy_db/files"
+
+  - id: files4
+    type: disk
+    store_by: id
+    name: "University of Freiburg storage (NetApp FabricPool, id=files4)"
+    weight: 0
+    files_dir: "/data/4/galaxy_db/files"
+
+  - id: files6
+    type: disk
+    store_by: id
+    name: "University of Freiburg storage (NetApp FabricPool, id=files6)"
+    weight: 0
+    files_dir: "/data/6/galaxy_db/files"
+
+  - id: files7
+    type: disk
+    store_by: id
+    name: "University of Freiburg storage (NetApp FabricPool, id=files7)"
+    weight: 0
+    files_dir: "/data/7/galaxy_db/files"    
+
+  - id: files8
+    type: disk
+    store_by: id
+    name: "University of Freiburg storage (NetApp FabricPool, id=files8)"
+    weight: 0
+    files_dir: "/data/dnb-ds01/galaxy_db/files"
+
+  - id: files9
+    type: disk
+    store_by: id
+    name: "University of Freiburg storage (NetApp FabricPool, id=files9)"
+    weight: 0
+    files_dir: "/data/dnb-ds02/galaxy_db/files"
+
+  - id: files10
+    type: disk
+    store_by: uuid
+    name: "University of Freiburg storage (NetApp FabricPool, id=files10)"
```

Comment:
```suggestion
    name: "University of Freiburg Isilon storage (id=files10)"
```

Reply:
We migrated the data I think. Is it really on Isilon still? I think it was all moved.

Reply:
Right!